src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1422,3 +1422,53 @@ function random_orientation_dag(
     end
     return g2
 end
+
+"""
+    bernoulli_graph(Λ)
+
+Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with probability ``\\operatorname{Bernoulli}(\\Lambda[i,j])``.
+"""
+function bernoulli_graph(
+    Λ::Matrix{Float64};
```

Comment:
```suggestion
    Λ::AbstractMatrix{<:AbstractFloat};
```
I would imagine that one wants to pass in a `SparseMatrixCSC` here.